Eventy/src/main/java/org/example/eventy/reviews/models/Review.java

```diff
@@ -0,0 +1,74 @@
+package org.example.eventy.reviews.models;
+
+import org.example.eventy.common.models.Status;
+import org.example.eventy.events.models.Event;
+import org.example.eventy.solutions.models.Service;
+
+public class Review {
+    private Event event;
+    private Service service;
+    private String comment;
+    private Integer grade;
+    private Status status;
+    private Boolean isDeleted;
```

Comment:
If Review has a logic delete, maybe deletion should be PUT in the controller? This is a good question, I am not sure too.

Reply:
We will ask this 😭